app/client/ui/AppUI.ts

```diff
@@ -83,15 +83,15 @@ function createMainPage(appModel: AppModel, appObj: App) {
     } else if (pageType === 'billing') {
       return domAsync(loadBillingPage().then(bp => dom.create(bp.BillingPage, appModel)));
     } else if (pageType === 'welcome') {
-      return dom.create(WelcomePage, appModel);
+      return dom.create(WelcomePage, appModel, appObj);
     } else if (pageType === 'account') {
-      return domAsync(loadAccountPage().then(ap => dom.create(ap.AccountPage, appModel)));
+      return domAsync(loadAccountPage().then(ap => dom.create(ap.AccountPage, appModel, appObj)));
     } else if (pageType === 'admin') {
-      return domAsync(loadAdminPanel().then(m => dom.create(m.AdminPanel, appModel)));
+      return domAsync(loadAdminPanel().then(m => dom.create(m.AdminPanel, appModel, appObj)));
     } else if (pageType === 'activation') {
       return domAsync(loadActivationPage().then(ap => dom.create(ap.getActivationPage(), appModel)));
     } else if (pageType === 'audit-logs') {
-      return domAsync(loadAuditLogsPage().then(m => dom.create(m.AuditLogsPage, appModel)));
+      return domAsync(loadAuditLogsPage().then(m => dom.create(m.AuditLogsPage, appModel, appObj)));
```

Comment:
I didn't update all the pages because it seems some are related to the EE edition but if this gets merged as is, they would need to be updated to benefit from the region focus switcher.